isc_dhcp_leases/iscdhcpleases.py

```diff
@@ -99,16 +100,19 @@ class IscDhcpLeases(object):
         r"ia-(?P<type>ta|na|pd) \"(?P<id>[^\"\\]*(?:\\.[^\"\\]*)*)\" {(?P<config>[\s\S]+?)\n}")
     regex_iaaddr = re.compile(r"ia(addr|prefix) (?P<ip>[0-9a-f:]+(/[0-9]+)?) {(?P<config>[\s\S]+?)\n\s+}")
 
-    def __init__(self, filename):
+    def __init__(self, filename, gz=False):
```

Comment:
the `gz` parameter isn't what you added to the readme, if you change this to `gzip=` instead then I will merge it and create a new release.